prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+"""
+Emails
+Estimate: 40 minutes
+Actual:  35  minutes
+"""
+
+
+def main():
+    """Print user's name and email"""
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        username = obtain_name_from_email(email)
+        enquiry = input(f"Is your name {username}? (Y/n) ").lower()
+        if enquiry != "y" and enquiry != "":
```

Comment:
It cannot work to full answer (e.g. yes/yea)

Reply:
Noted. For me, this code is written with the assumption that the user's input can only be Y or enter. Thank you for comment!

Reply:
Good. But be careful.